modules/readme.js

```diff
@@ -0,0 +1,67 @@
+//function to generate the readme file
+
+//Object Destructuring
+const createReadme = (data) => {
+  const {
```

Comment:
nice destructing!
Try to use this when the # of vals is fairly small i.e. 4 for better readability, but never the less, great work :-)